### ISSUE TITLE
Dependency on armeabi-v7a reports an error

### ISSUE THREAD

**Issue by cl-6666** [gkonovalov/android-vad#30]
When the main project depends on armeabi-v7a, it crashes directly. Silero is used.
ndk {
abiFilters "armeabi-v7a"
}

**Comment by gkonovalov** [comment]
You need specify next code:

` defaultConfig {
        ndk {
            abiFilters  "armeabi-v7a"
        }
    }`

**Comment by cl-6666** [comment]
I tried the method you mentioned above, but it still crashed. My gradle configuration is as follows
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/0ef6b4b3-3c31-48ca-a12b-dbe74231d3fd" />
<img width="810" alt="image" src="https://github.com/user-attachments/assets/ea15304a-0569-45b2-a697-a14dc8a2d3c9" />
<img width="870" alt="image" src="https://github.com/user-attachments/assets/b8a01c0e-1545-4d20-88ef-900bd971f066" />
<img width="862" alt="image" src="https://github.com/user-attachments/assets/308a45de-dc6b-400a-931f-b6cbd7c321c7" />
依赖版本是这个：    implementation 'com.github.gkonovalov.android-vad:silero:2.0.7'

**Comment by gkonovalov** [comment]
I will try to reproduce it from my side, but looks like crash in ONNX Runtime library.

**Comment by gkonovalov** [comment]
I see you are using old Gradle version. You probably need to update it.

**Comment by cl-6666** [comment]
> 我发现你正在使用旧版 Gradle。你可能需要更新它。

Because the project is relatively old, is it compatible with the old Gradle version? I found that there is no problem using this architecture ndk {abiFilters "arm64-v8a"}, but using this architecture will report an error ndk {abiFilters "armeabi-v7a"}

**Comment by gavine99** [comment]
i had the same issue ... this fixed it for me; https://onnxruntime.ai/docs/build/android.html#note-proguard-rules-for-r8-minimization-android-app-builds-to-work.
TLDR; add "-keep class ai.onnxruntime.** { *; }" to proguard-rules.pro file to stop selected entry points being minified away when 'minifyEnabled true' in release builds.

**Comment by gkonovalov** [comment]
Thanks @gavine99! 
@cl-6666 I included ProGuard rules in v2.0.8 library release, so it should fix this issue.
